tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/ICreateQueueRequest.cs

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
```

Comment:
To clarify, given it's much more normal for list properties to be explicitly initialized with non-null values, I've been less concerned about marking those nullable personally